crates/subspace-farmer/src/plot/piece_index_hash_to_offset_db.rs

```diff
@@ -200,7 +208,7 @@ impl IndexHashToOffsetDB {
 
     pub(super) fn batch_insert(
         &mut self,
-        index_hashes: &[PieceIndexHash],
+        index_hashes: Vec<PieceIndexHash>,
```

Comment:
:thinking: isn't `batch_insert` and `batch_put` are kinda duplicates?

Reply:
`batch_insert` is used to insert new entries and keeps track of number of inserted pieces. `batch_put` is called from two places, one of which is `replace_furthest`, which doesn't affect number of pieces stored.